packages/cli/src/spec/mockPromptSpec.ts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Can we rename this file to `MockPromptSpec.ts` (with capital M). Also, let's ensure this is consistent to the name for mockprompt (currently `MockPromptForStrings`).

Reply:
So, should I call it `MockPromptForStringsSpec.ts`? (assuming we don't change the classes name)